content/merchants/index.md

```diff
@@ -20,6 +20,20 @@ type: "default"
 <p>If your store accepts NavCoin and you would like to be listed on this page, read about how you can contribute to this website.
 {{< /left_image_section >}}
 
```

Comment:
try removing this empty line to fix the empty ```<p>``` tag causing that white space between sections